platform/web-security/src/main/java/org/motechproject/security/passwordreminder/PasswordExpirationCheckerInternal.java

```diff
@@ -0,0 +1,30 @@
+package org.motechproject.security.passwordreminder;
+
+import org.motechproject.event.MotechEvent;
+import org.motechproject.scheduler.contract.CronSchedulableJob;
+import org.motechproject.scheduler.service.MotechSchedulerService;
+
+import static org.motechproject.security.constants.EventSubjects.PASSWORD_EXPIRATION_CHECK;
+
+/**
+ * Responsible for scheduling job that will
```

Comment:
Mention in this and the previous the reason for all this mumbo jumbo - having an optional dependency on the scheduler module OSGi-wise